<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpApp</class>
 <widget class="QMainWindow" name="TcpApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>4</y>
      <width>414</width>
      <height>456</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="serverRB">
         <property name="text">
          <string>TCP服务器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="clientRB">
         <property name="text">
          <string>TCP客户端</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IpEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>请输入ip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PortEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>请输入端口号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartBt">
         <property name="text">
          <string>开启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBt">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>在线：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="onlineUserCout">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_open">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="onlineUserList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="recieveList"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>接收：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="receiveNumLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearRcvBt">
           <property name="text">
            <string>清空接收区</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>发送：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sendNumLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearSendBt">
           <property name="text">
            <string>清空发送区</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="sendMsgEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="autoTimeEidt">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>请输入时间值ms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="autoCB">
           <property name="text">
            <string>自动</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sendMsgBt">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
